great_expectations/rule_based_profiler/data_assistant/data_assistant_dispatcher.py

```diff
@@ -1,21 +1,21 @@
+import logging
 from typing import Dict, Optional, Type
 
-from great_expectations.expectations.registry import get_data_assistant_impl
 from great_expectations.rule_based_profiler.data_assistant import DataAssistant
 from great_expectations.rule_based_profiler.data_assistant.data_assistant_runner import (
     DataAssistantRunner,
 )
 
+logger = logging.getLogger(__name__)
+
 
 class DataAssistantDispatcher:
     """
     DataAssistantDispatcher intercepts requests for "DataAssistant" classes by their registered names and manages their
     associated "DataAssistantRunner" objects, which process invocations of calls to "DataAssistant" "run()" methods.
     """
 
-    DATA_ASSISTANT_TYPE_ALIASES: Dict[str, str] = {
-        "volume": "volume_data_assistant",
-    }
+    _registered_data_assistants: Dict[str, Type[DataAssistant]] = {}
```

Comment:
This is truly private now - the dispatcher acts as the interface callers need to interact with to use the values contained herein.